packages/core-data/src/entities.js

```diff
@@ -373,6 +373,7 @@ async function loadPostTypeEntities() {
 					 * @type {Record< string, boolean >}
 					 */
 					supports: {
+						awareness: true,
```

Comment:
Why is awareness a properly of the entity config. It feels to me like it's something global to the whole sync manager no?